### PR TITLE
Hw 03 spring

### PR DESCRIPTION
**Pull request by ivlytvynets**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,6 +56,33 @@
...
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
+            <version>2.3.3</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
+            <version>1.2.2</version>
+        </dependency>
```
Comment (Likh-Alex) [inline pom.xml]: we can delete jsp dependencies as we are passing json to the page, take a look in the checklist

**Review by Likh-Alex — APPROVED** [review]

File: src/main/java/com/dev/lytvynets/dto/UserResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.dev.lytvynets.dto;
+
+public class UserResponseDto {
+    private String firstName;
```
Comment (Sofasmile) [inline src/main/java/com/dev/lytvynets/dto/UserResponseDto.java:5]: let's add id here too

File: src/main/java/com/dev/lytvynets/service/UserService.java [hunk 3]
```diff
@@ -2,9 +2,12 @@
 
 import com.dev.lytvynets.model.User;
 import java.util.List;
+import java.util.Optional;
 
 public interface UserService {
     void add(User user);
 
+    Optional<User> get(Long id);
```
Comment (Sofasmile) [inline src/main/java/com/dev/lytvynets/service/UserService.java]:
```suggestion
    User get(Long id);
```

File: src/main/java/com/dev/lytvynets/service/impl/UserMapperImpl.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.dev.lytvynets.dto.UserResponseDto;
+import com.dev.lytvynets.model.User;
+import com.dev.lytvynets.service.UserMapper;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserMapperImpl implements UserMapper {
+    @Override
+    public User getUserEntity(UserResponseDto userResponseDto) {
```
Comment (Sofasmile) [inline src/main/java/com/dev/lytvynets/service/impl/UserMapperImpl.java]: where do we use this method? Let's remove it)

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**